source/documentation/metrics-install-guide/Setting_Up_OpenShift_Aggregated_Logging.html.md

```diff
@@ -77,11 +77,19 @@ For example:
 
 The installer for OpenShift Container Platform is provided by the `atomic-openshift-utils` package.
```

Comment:
Well I really do need to update the installation. In this part , under "## Installing OpenShift Aggregated Logging Packages" I would only refer to follow the steps here:
https://docs.okd.io/latest/install/host_preparation.html#installing-base-packages
And perhaps only ass the 

Install centos-release-ovirt42 to get an updated ansible runtime
    # yum install centos-release-ovirt42

Do you want me to update it or will you ? @sandrobonazzola

Reply:
please go ahead updating it, you know the argument better than me

Reply:
+1

Reply:
If documentation team added these files can I remove the files under ./source/develop/release-management/features/metrics ?

@sandrobonazzola